nautobot/extras/filters.py

```diff
@@ -105,18 +106,20 @@ class CreatedUpdatedFilterSet(django_filters.FilterSet):
 class ComputedFieldFilterSet(BaseFilterSet):
     q = SearchFilter(
         filter_predicates={
-            "name": "icontains",
-            "target_url": "icontains",
-            "text": "icontains",
+            "label": "icontains",
+            "description": "icontains",
             "content_type__app_label": "icontains",
             "content_type__model": "icontains",
+            "template": "icontains",
+            "fallback_value": "icontains",
```

Comment:
These were just plain wrong before, but we didn't have a ComputedField filters test case to detect that (which I've now added)